src/Serializer/Tests/AbstractItemNormalizerTest.php

```diff
@@ -0,0 +1,1338 @@
+<?php
```

Comment:
This should be moved from `tests/`, you should probably remove them from `tests/`